lib/acl-checker.js

```diff
@@ -36,13 +35,10 @@ class ACLChecker {
     // Check the resource's permissions
     return this._permissionSet
       .then(acls => this.checkAccess(acls, user, mode))
-      .catch(err => {
-        debug(`Error: ${err.message}`)
+      .catch(() => {
         if (!user) {
-          debug('Authentication required')
           throw new HTTPError(401, `Access to ${this.resource} requires authorization`)
         } else {
-          debug(`${mode} access denied for ${user}`)
```

Comment:
This seems like a useful message, no?

Reply:
It is, just not 8 times :wink: That's why I moved it here: https://github.com/solid/node-solid-server/pull/566/files#diff-b11433f1fb19c85e170915d27502d490R43